src/components/checkout/PlatformCreditCheckoutSection.test.tsx

```
@@ -0,0 +1,62 @@
+    expect(screen.getByText("$0.00")).toBeInTheDocument();
+    expect(screen.getByText(/No credit card required/i)).toBeInTheDocument();
+  });
+
+  it("renders partial deduction when balance is less than order total", () => {
+    // Balance is $20.00 (2000 cents), Order is $50.00 (5000 cents)
+    render(
+      <PlatformCreditCheckoutSection
+        orderTotalCents={5000}
+        customBalanceCents={2000}
+      />,
+    );
```

ESLint

[error] Replace `⏎······<PlatformCreditCheckoutSection⏎········orderTotalCents={5000}⏎········customBalanceCents={2000}⏎······/>,⏎····` with `<PlatformCreditCheckoutSection·orderTotalCents={5000}·customBalanceCents={2000}·/>`

(prettier/prettier)